lm-feedUI-iOS/lm-feedUI-iOS/Source/Classes/UI/PostDetailWidgets/PostHeaderFooter/LMFeedPostDetailFooterView.swift

```diff
@@ -44,7 +44,7 @@ open class LMFeedPostDetailFooterView: LMFeedPostFooterView {
     
     open private(set) lazy var noCommentSubtitleLabel: LMLabel = {
         let label = LMLabel().translatesAutoresizingMaskIntoConstraints()
-        label.text = "Be the first one to comment"
+        label.text = "Be the first one to create a comment"
```

Comment:
Can we add this string in string constant file? Same apply for all static strings.